src/settings/browser-settings.js

```diff
@@ -262,11 +276,24 @@ const Settings = () => {
           </div>
         </div>
       </nav>
+      <nav class="panel">
+        <p class="panel-heading">Global notifications</p>
```

Comment:
Let's use this container as the container for the other settings
```suggestion
        <p class="panel-heading">Other settings</p>
```

Reply:
Done